### ISSUE TITLE
problem regarding  github repo...... 

### ISSUE THREAD

**Issue by RakeshY008** [dbc2201/gla-bcsc0001-2018#34]
sir can we upload pdf or other file containg images along text .........."https://github.com/RakeshY008/gla2k18/tree/master/Super%20Practice%20Set%202" 

see the solution sub-folder .....hope you will get to know what i am trying to ask..

**Comment by dbc2201** [comment]
Yes @RakeshY008 , you can, just select them while choosing the files to upload.

**Comment by RakeshY008** [comment]
> Yes @RakeshY008 , you can, just select them while choosing the files to upload.

> @dbc2201 , thank you sir!
